### ISSUE TITLE
backupccl: test flakes in the Bazel-CI target 

### ISSUE THREAD

**Issue by knz** [cockroachdb/cockroach#67480]
**Describe the problem**

The Bazel-CI target fails on the `backupccl` while the Github-CI target succeeds.

Unfortunately not much details are available:
![image](https://user-images.githubusercontent.com/642886/125299535-89ccd980-e329-11eb-9963-49081eb08f55.png)

The exit code indicates the program stopped with SIGALRM, which in turn indicates a timeout.

I tried reproducing this locally and the test passes, albeit taking a lot of time. I suspect the timeout in bazel is too low.

Alternatively perhaps this package could be broken up into more sub-packages with finer grained tests.

**Comment by rickystewart** [comment]
Yeah, it's [a timeout](https://teamcity.cockroachdb.com/repository/download/Cockroach_UnitTests_BazelUnitTests/3172184:id/bazel-testlogs/pkg/ccl/backupccl/backupccl_test/test.log).

    -- Test timed out at 2021-07-12 09:29:53 UTC --
